content/en/profiler/enabling/go.md

```diff
@@ -31,23 +31,25 @@ Continuous Profiler is not supported on some serverless platforms, such as AWS L
 
 ## Installation
 
+If you are using version 2.x of the Go Tracer, follow the import instrumentions marked with `2.x`. Profiler documentation for `dd-trace-go` version 2.x can be found [here][21].
+
 To begin profiling applications:
 
 1. Ensure Datadog Agent v6+ is installed and running. Datadog recommends using [Datadog Agent v7+][19].
 
 2. Get `dd-trace-go` using the command:
 
     ```shell
-    go get gopkg.in/DataDog/dd-trace-go.v1/profiler # v1
-    # go get github.com/DataDog/dd-trace-go/v2/profiler # v2
+    go get gopkg.in/DataDog/dd-trace-go.v1/profiler # 1.x
+    # go get github.com/DataDog/dd-trace-go/v2/profiler # 2.x
```

Comment:
Why 1.x and 2.x? The old v1 or v2 comments were fine IMO.

Reply:
I wanted to keep the comments consistent between here and the code on lines 51/52 so that the new instructions at the top would make more sense. I'm open to changing both to either v1/v2 or 1.x/2.x; what do you think?